inspire/modules/authors/config.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+## This file is part of INSPIRE.
+## Copyright (C) 2014, 2015 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+AUTHORS_IDENTIFIERS_TYPES = ['authorid', 'orcid', 'arxiv']
+
+INSPIRE_EMAIL_REGEX = "^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$"
+INSPIRE_URL_REGEX = "^(?:http(?:s)?:\/\/)?(?:www\.)?(?:[\w-]*)\.\w{2,}$"
+
+INSPIRE_AFFILIATION_RANKS = ['senior', 'junior', 'staff', 'visitor', 'postdoc',
+                             'phd', 'masters', 'undergrad']
+
+# year - month (2 digits) -day (2 digits)
+INSPIRE_DATE_REGEX = "^[1|2][0-9]{3}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0|1])$"
+
+INSPIRE_AFFILIATION_STATUS = ['current']
+INSPIRE_EXPERIMENT_STATUS = ['current']
+INSPIRE_EMAIL_STATUS = ['current']
+
+INSPIRE_NAME_NUMERATIONS = "^Jr\.|Sr\.|III|IV|VI{0,3}$"
+INSPIRE_NAME_STATUSES = ['active', 'deceased', 'departed', 'retired']
+INSPIRE_NAME_TITLES = ['Sir']
+
+
+INSPIRE_YEAR_MIN = 1900
```

Comment:
It export a `datetime` class whose implementation dosen't suffer from Python native's one. See: 8a876e573432c46ac27084ea080c94855e4fe1c0

In particular since I see `INSPIRE_YEAR_MIN = 1900` and we surely have papers dating back to 1700.